lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize(filename)
+    @dishes = {}
+    load_menu(filename)
+  end
+
+  def read_menu
+    @dishes
+  end
+
+  private
+
+  def load_menu(filename)
```

Comment:
This is a lovely neat method.  Could the code use inject to build the @dishes? This would make the iteration more functional eg:

```
@dishes = file.readlines.inject({})...
```